src/main/kotlin/nomic/api/RulesAmendments.kt

```diff
@@ -0,0 +1,42 @@
+package nomic.api
+
+import nomic.data.EntityNotFoundException
+import nomic.domain.entities.RulesAmendmentsModel
+import nomic.domain.rulesamendments.RuleAmendmentDomain
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("api")
+class RulesAmendments(val ruleAmendmentDomain: RuleAmendmentDomain) {
+    // Path to endpoint is api/rules_amendments/ExistingGameId
+    @GetMapping("rules_amendments/{gameid}")
+    fun getRulesAmendments(@PathVariable(value = "gameid") gameId: String): ResponseEntity<Any> {
+        val responseHeader = HttpHeaders()
+        responseHeader.set("Content-Type", "application/json")
+        responseHeader.set("charset", "UTF-8")
+
+        val rulesAmendments: MutableList<RulesAmendmentsModel>
+
+        try {
```

Comment:
Why do you use try catch? Couldn't you more gracefully handle errors where possible and harness Spring's exception handling to take care of other errors outside of the endpoint? Otherwise, it seems like this kind of code will get replicated across many endpoints.